update-changelog/index.ts

```diff
@@ -119,6 +121,29 @@ class UpdateChangelog extends Action {
 				labels: ['backport ' + versionMajorBranch, 'no-changelog'],
 			})
 		}
+
+		// publish changelog to community.grafana.com
+		const forumData = {
```

Comment:
nit: `forumData` can be moved into the `if(apiKey)`

Reply:
moved. ty!